docs/intro-to-coalton.md

```diff
@@ -192,9 +196,32 @@ There are convenient *syntaxes* for composing functions with the
 
 These are useful to make code less noisy.
 
-Note that since these are macros (indicated by their variadic arguments), they
-cannot be used as higher-order functions. Consider either currying or the
-`compose` function if you're thinking in that direction.
+Note that since these are macros (indicated by their variadic arguments), they cannot be used as higher-order functions. Consider either currying or the `compose` function if you're thinking in that direction.
+
+### Ignoring Function Parameters
+
+Consider the function
+
+```lisp
+(coalton-toplevel
+  (define (f x y)
+    x))
+```
+
+Here, `y` is unused and will produce a warning:
+
+```
+COMMON-LISP:WARNING: warn: Unused variable
+  --> COALTON-TOPLEVEL (NIL):1:31
+   |
+ 1 |  (COALTON-TOPLEVEL (DEFINE (F X Y)
+   |                                 ^ variable defined here
+help: prefix the variable with '_' to decare it unused
+ 1 | (COALTON-TOPLEVEL (DEFINE (F X _Y)
+   |                                --
+```
+
+As suggested, one can replace `y` with `_y`, which tells the Coalton compiler that the parameter might be intentionally unused.
```

Comment:
Should we document that it's still possible to use `_y`, though users are discouraged from doing that except in corner cases like macros? Something like:

> Variables prefixed with `_` like `_y` are still normal variables, and can be read. The following is valid Coalton:
> ```
> (define (f _x) _x)
> ```
> You should treat underscore prefixed variables as ignored whenever possible, and use a name not prefixed with `_` if you intend to use it. Reading from underscore-prefixed variables is permitted so that generated code (e.g. using macros or read-conditionals) may avoid unused variable warnings for variables which will be used in some compilation contexts but not others.